### PR TITLE
WIP: add Merge When Ready to app directory

### PR DESCRIPTION
**Pull request by cyberhck**
##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [ ] All comments in frontmatter need to be removed. (Don't know what this is)
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: {automatically merges your PR when checks pass}
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Has tests](https://github.com/fossapps/MergeWhenReady)
- [x] [Has documentation](https://github.com/fossapps/MergeWhenReady/) (it's a typescript app)
- [x] [Is open source](https://github.com/fossapps/MergeWhenReady)
- [x] [Has a license](https://github.com/fossapps/MergeWhenReady/blob/master/LICENSE.md)
- [x] [Has a code of conduct](https://github.com/fossapps/MergeWhenReady/blob/master/CODE_OF_CONDUCT.md)
- [x] Has someone willing to at least minimally maintain them for the near future: {Nishchal Gautam (@cyberhck)}

-----
[View rendered _apps/merge-when-ready.md](https://github.com/cyberhck/probot.github.io/blob/patch-2/_apps/merge-when-ready.md)

### CODE REVIEW

**Comment by cyberhck** [comment]
working on it

File: _apps/merge-when-ready.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: Merge When Ready
+description: Add a special label to merge your PR when status checks pass
+slug: mergewhenready
+screenshots:
+- https://marketplace-screenshots.githubusercontent.com/4760/cb554780-a829-11e9-8993-28186c68aaab
+- https://marketplace-screenshots.githubusercontent.com/4760/dd36ea80-a829-11e9-8fc4-3571da4aa9f0
+- https://marketplace-screenshots.githubusercontent.com/4760/ed4eca00-a829-11e9-9beb-59766847ccd5
+authors:
+- cyberhck
+repository: fossapps/Handlebars-Issue-and-Pull-Requests
```
Comment (gr2m) [inline _apps/merge-when-ready.md]: looks like the wrong repository?

Comment (cyberhck) [reply _apps/merge-when-ready.md]: yes, and also, the app didn't work the way I expected, I need to update, but haven't gotten any time to do so, will try to do soon :slightly_smiling_face:

Comment (gr2m) [reply _apps/merge-when-ready.md]: Shall we close the PR for the time being and you reopen it once you have something ready?

**Pull request closed without merging**